changelog.md

```diff
@@ -70,6 +71,7 @@
 
 * [1391](https://github.com/zeta-chain/node/pull/1391) - consolidate node builds
 * update `MsgUpdateContractBytecode` to use code hash instead of contract address
+>>>>>>> develop
```

Comment:
i think you wanna remove this

Reply:
[d05b9b0](https://github.com/zeta-chain/node/pull/1436/commits/d05b9b0c8c4f42d840f835bb7984384bd6f9db22)